### PR TITLE
Added refresh token

### PR DESCRIPTION
**Pull request by Habiba67y**

### CODE REVIEW

**Review by Abdurasulovich — COMMENTED** [review]
Done

**Review by sulton-max — DISMISSED** [review]
Looks good to me

File: src/Training.AirBnb.Clone.Backend/AirBnB.Api/Models/DTOs/IdentityTokenDto.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+namespace AirBnB.Api.Models.DTOs;
+
+public class IdentityTokenDto
+{
+    public string AccessToken { get; set; } = default!;
+
+    public string RefreshToken { get; set; } = default!;
+}
```
Comment (Ilxom-22) [inline src/Training.AirBnb.Clone.Backend/AirBnB.Api/Models/DTOs/IdentityTokenDto.cs:8]: Comments needed here

**Review by Ilxom-22 — CHANGES_REQUESTED** [review]

File: src/Training.AirBnb.Clone.Backend/AirBnB.Domain/Extension/ExceptionExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        {
+            result = new FuncResult<T>(await func());
+        }
+        catch (Exception e)
+        {
+            result = new FuncResult<T>(e);
+        }
+
+        return result;
+    }
```
[the remark this answers is not in the document]
Comment (Habiba67y) [reply src/Training.AirBnb.Clone.Backend/AirBnB.Domain/Extension/ExceptionExtensions.cs:53]: aha, got it

File: src/Training.AirBnb.Clone.Backend/AirBnB.Api/Models/DTOs/IdentityTokenDto.cs (hunk 1 above)
Comment (Habiba67y) [reply src/Training.AirBnb.Clone.Backend/AirBnB.Api/Models/DTOs/IdentityTokenDto.cs:8]: ok

**Review by Ilxom-22 — APPROVED** [review]

**Pull request merged**